runtime/gc_realtime/MetronomeAlarm.hpp

```diff
@@ -99,8 +98,10 @@ class MM_HRTAlarm : public MM_Alarm
 
 class MM_RTCAlarm : public MM_Alarm
 {
-private:	
+private:
+#if defined(LINUX) && !defined(J9ZTPF)
 	IDATA RTCfd;
+#endif /* defined(LINUX) && !defined(J9ZTPF) */
```

Comment:
Would you please also add "&& !defined(J9ZTPF)" to MM_RTCAlarm::sleep() in MetronomeAlarm.cpp for consistency ?

Reply:
fixed.